images/httpd/Dockerfile

```diff
@@ -1,30 +0,0 @@
-#-------------------------------------------------------------------------------
```

Comment:
we use this image in https://github.com/gravitee-io/gravitee.io/blob/master/Dockerfile, perhaps we should move to nginx